src/components/common/PostalCode/PostalCodeBar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Button from 'src/components/ui/Button'
+import Icon from 'src/components/ui/Icon'
+import './postal-code.scss'
+
+interface PostalCodeBarProps {
+  content?: string
+}
+
+export default function PostalCodeInput({ content }: PostalCodeBarProps) {
+  return (
+    <div data-postal-code-bar>
+      <Button>
```

Comment:
```suggestion
      <Button 
            variant="tertiary"
            icon={<Icon name="MapPin" width={24} height={24} />}
            iconPosition="left"
        >
```

Maybe you can try using the tertiary variant here 🤔